packages/atomic/src/components/facets/facet/base-facet.pcss

```diff
@@ -1,5 +1,5 @@
 .content {
-  z-index: 1000;
+  z-index: 1;
```

Comment:
What was the reason for this value change? Is the z-index set on the modal that opens in responsive mode?

Reply:
Yeah it's for the modal for the responsive facet, I was fiddling around with the `z-index` in order to get the best result for accessibility, but I actually don't think this change is needed (I initially set it to 1000 so that way it would go over any of the end users elements as well). So I'll revert it.

Reply:
Could we use a tailwind class instead of a z-index in the css?

Reply:
^ Also, as far as value, we should use the smallest necessary number.

We actually do not want to go over end user's elements. For example, let's say an e-commerce customer wants to display a modal while the shopper is looking at the facet values on a mobile device, their modal should appear over the values.

If it does not, the customer's developers will simply use a higher z-index to force it (e.g. `1001`). There is no way force our elements to be over our customers', but also we do not want them to be. We should use just the minimum value necessary to make our components work correctly.